lab-mikeporter/route/noteRouter.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const uuid = require('uuid');
+const router = require('../lib/router.js');
+const Note = require('../model/note.js');
+
+router.post('/api/notes', (req, res) => {
+  if (!req.body.content) return res.sendStatus(400);
+
+  let id = uuid.v1();
+  let date = new Date();
+  let content = req.body.content;
+  new Note(id, date, content)
+    .save()
+    .then((note) => res.sendJSON(201, note))
+    .catch((err) => res.sendStatus(500));
```

Comment:
Running your npm lint script brought up some errors of unused variables. Here you're invoking a function in the catch block with an argument 'err' but then not using it. Best practice would be to either invoke the function without an argument () => OR pass the err.status to the response object.